### PR TITLE
fix(Carousel): fixes #4136 - reverse carousel slide navigation direction for touch

### LINKED ISSUE #4136: Carousel slide navigation direction on swipe is incorrect
## Describe the bug

React-bootstrap beta 10 version added touch support in Carousel Component, but the direction of slide navigation is right on right swipe and left on left swipe.

## To Reproduce

1. Go to https://react-bootstrap.github.io/components/carousel/ on mobile or tablet or other touch device.
2. Try Swiping on the demo slides.

## Expected behavior

The carousel should navigate to previous slide on right swipe and next slide on left swipe.

## Environment (please complete the following information)

- Operating System: MacOS
- Browser Version: Chrome 73
- React-Bootstrap: Version 1.0.0-beta.10

### PR DESCRIPTION
**Pull request by RageJS**
See issue [4136](https://github.com/react-bootstrap/react-bootstrap/issues/4136)

### CODE REVIEW

File: src/Carousel.js [hunk 1]
```diff
@@ -242,11 +242,13 @@ class Carousel extends React.Component {
     )
       return;
 
-    // Going left
-    if (e.changedTouches[0].screenX < this.state.touchStartX)
+    if (e.changedTouches[0].screenX < this.state.touchStartX) {
+      // Going right
```
Comment (taion) [inline src/Carousel.js]:
```suggestion
      // Swiping left to navigate to next item.
```

File: src/Carousel.js [hunk 2]
```diff
@@ -242,11 +242,13 @@ class Carousel extends React.Component {
...
+      this.handleNext(e);
+    } else {
+      // Going left
```
Comment (taion) [inline src/Carousel.js]:
```suggestion
      // Swiping right to navigate to previous item.
```

**Comment by taion** [comment]
The changes are fine, but please accept the updates to the comments, and fix the code formatting so CI passes.

**Comment by RageJS** [comment]
@taion Made the changes as per PR suggestions.

**Comment by bpas247** [comment]
https://github.com/react-bootstrap/react-bootstrap/blob/fce0e8b3e12a9add158befe3548fd1716e2687c7/test/CarouselSpec.js#L308-L332
I'm thinking we'll need to fix these tests so that they're actually testing if it's swiping left or right. I think right now it's only really testing if it changes to the other `Carousel.Item` component or not when it's swiped (since there are only two items in the Carousel for those tests).

**Comment by RageJS** [comment]
@bpas247 I think the code you're checking, that's from current master branch. In my PR, I had changed that file to have three carousel items, and checked swipe direction accordingly. Can be checked here, https://github.com/react-bootstrap/react-bootstrap/pull/4137/files

**Pull request merged**